pytest_fxa/plugin.py

```diff
@@ -48,12 +48,27 @@ def pytest_generate_tests(metafunc):
         metafunc.parametrize('fxa_urls', envs, indirect=True)
 
 
-@pytest.fixture
+def fxa_cleanup(account, fxa_client, fxa_account):
+    logger = logging.getLogger()
+    try:
+        account.clear()
+        fxa_client.destroy_account(fxa_account.email, fxa_account.password)
+        logger.info('Removed: {}'.format(fxa_account))
+    except ClientError as e:
+        # 'Unknown Account' error is ok - account already deleted
+        # https://github.com/mozilla/fxa-auth-server/blob/master/docs/api.md#response-format
+        if e.errno not in [102]:
+            raise
+    finally:
+        return
+
+
+@pytest.fixture(scope='session')
```

Comment:
Is there a reason we're making this a session scoped fixture? Sorry if this has already been discussed, I've been away for a week and seem to have forgotten a lot!

Reply:
That was done in error.